packages/cli/src/commands/__tests__/lint.test.ts

```diff
@@ -204,7 +207,7 @@ describe('lint', () => {
   });
 
   it('shows help if unknown format is passed', () => {
```

Comment:
Perhaps pop a test in here to verify something like `spectral lint --format stylish --format junit --output.junit junit-results.xml my-api.yaml` doesn't regress in future?

Reply:
we have it already haha :D
See https://github.com/stoplightio/spectral/issues/2042#issuecomment-1026193383

When I made the following change https://github.com/stoplightio/spectral/pull/2044/files#diff-5c1bab693e4315f93ddc3bc536502fe23b536bbed6214ca5a093571db6389f24R14-R21, the test started to fail.